tests/test_data/isb_core_csv_documents/simple_isamples.csv

```diff
@@ -0,0 +1,2 @@
+id	label	description	hasContextCategory	hasMaterialCategory	hasSpecimenCategory	keywords	informalClassification	producedBy_label	producedBy_description	producedBy_hasFeatureOfInterest	producedBy_responsibility	producedBy_resultTime	producedBy_samplingSite_description	producedBy_samplingSite_label	producedBy_samplingSite_location_elevationInMeters	producedBy_samplingSite_location_latitude	producedBy_samplingSite_location_longitude	producedBy_samplingSite_placeName	registrant	samplingPurpose	curation_label	curation_description	curation_accessContraints	curation_location	curation_responsibility	compliesWith	authorizedBy
+DOI:12345	Label 12345	This is a description of sample 12345.	["context1"]	["material1"]	["specimen1"]	["keyword1"]	["Homo sapiens"]	A label for the event that produced the sample.	A description of the event that produced the sample.	Something interesting about what produced the sample.	["Who is responsible for producing the sample"]	2022-01-01T00:00:00Z	A description of the sampling site.	The sampling site label.	2792	32.4434174	-110.7881222	["Geographical name 1", "Geographical name 2"]	["Sample registrant 1"]	["Sampling purpose 1"]	A special label for curation.	A description of how the sample is curated.	Any constraints associated with curation.	Where the sample is curated.	The institution responsible for curation.	["URI 1 to compliance documentation"]	["URI 1 to associated permit"]
```

Comment:
Mostly just tab-separated strings -- exceptions are dates, numbers, and multi-valued strings, where we'll accept JSON arrays.  We could use a quote separator, but given that I knew we were going to embed multi-valued quoted string arrays as values, it seemed cleaner to use tabs.

Reply:
The CSV module has a way to do custom dialects -- going to see if I can specify formats for the typed data and have it emit errors if not.